docs/lang/basic.md

```diff
@@ -12,137 +12,136 @@ int main() {
 }
 ```
 
-??? note "什么是 include？"
-     `#include` 其实是一个预处理命令，意思为将一个文件“放”在这条语句处，被“放”的文件被称为头文件。也就是说，在编译时，编译器会“复制”头文件 `iostream` 中的内容，“粘贴”到 `#include <iostream>` 这条语句处。这样，你就可以使用 `iostream` 中提供的 `std::cin` 、 `std::cout` 、 `std::endl` 等对象了。
+??? note "What is include？"
+     `#include` is actually a preprocessor command, which means to "put" a file at this statement. The file being "put" is called a header file. In other words, when compiling, the compiler will "copy" the content in the header file `iostream` and "paste" it to the statement `#include <iostream>`. In this way, you can use the `std::cin`, `std::cout`, `std::endl` and other objects provided by `iostream`.
 
-    如果你学过 C 语言，你会发现目前我们接触的 C++ 中的头文件一般都不带 `.h` 后缀，而那些 C 语言中的头文件 `xx.h` 都变成了 `cxx` ，如 `stdio.h` 变成了 `cstdio` 。因为 C++ 为了和 C 保持兼容，都直接使用了 C 语言中的头文件，为了区分 C++ 的头文件和 C 的头文件，使用了 `c` 前缀。
+    If you have studied the C language, you will find that the header files in C++ that we are currently exposed to generally do not have the suffix `.h`, and those header files in the C language `xx.h` have become `cxx`. For example, `stdio.h` becomes `cstdio`. To maintain compatibility with C, C++ directly uses the header files in the C language. In order to distinguish between C++ header files and C header files, the `c` prefix is used.
       
-    一般来说，应当根据你需要编写的 C++ 程序的需要来确定你要 `#include` 哪些头文件。但如果你 `#include` 了多余的头文件，只会增加编译时间，几乎不会对运行时间造成影响。目前我们只接触到了 `iostream` 和 `cstdio` 两个头文件，如果你只需要 `scanf` 和 `printf`，就可以不用 `#include <iostream>` 。
+    Generally speaking, you should determine which header files you want to `#include` according to the C++ program you need to write. But if you `#include` the extra header files, it will only increase the compilation time and will hardly affect the runtime. At present, we have only come into contact with the two header files `iostream` and `cstdio`. If you only need `scanf` and `printf`, then it is not needed to use `#include <iostream>`.
 
-    可以 `#include` 自己写的头文件吗？答案是，可以。
+    Can I `#include` the header files written by myself? Yes.
 
-    你可以自己写一个头文件，如： `myheader.h` 。然后，将其放到和你的代码相同的目录里，再 `#include "myheader.h"` 即可。需要注意的是，自定义的头文件需要使用引号而非尖括号。当然，你也可以使用编译命令 `-I <header_file_path>` 来告诉编译器在哪找头文件，就不需要将头文件放到和代码相同的目录里了。
+    You can write a header file yourself, such as: `myheader.h`. Then, put it in the same directory as your code, and then `#include "myheader.h"`. It should be noted that the custom header files need to use quotation marks instead of brackets. Of course, you can also use the compilation command `-I <header_file_path>` to tell the compiler where to find the header file, so you don't need to put the header file in the same directory as the code.
```

Comment:
```suggestion
    You can write a header file like `myheader.h` yourself. Then, put it in the same directory as your code, and then `#include "myheader.h"`. It should be noted that the custom header files need to use quotation marks instead of brackets. Of course, you can also use the compilation command `-I <header_file_path>` to tell the compiler where to find the header file, so you don't need to put the header file in the same directory as the code.
```